src/context/FoxEthProvider/FoxEthProvider.tsx

```diff
@@ -126,11 +132,34 @@ export const FoxEthProvider = ({ children }: FoxEthProviderProps) => {
   const ethMarketData = useAppSelector(state => selectMarketDataById(state, ethAssetId))
   const foxMarketData = useAppSelector(state => selectMarketDataById(state, foxAssetId))
   const foxAssetPrecision = useAppSelector(state => selectAssetById(state, foxAssetId)).precision
+  const ethAssetPrecision = useAppSelector(state => selectAssetById(state, ethAssetId)).precision
   const lpAssetPrecision = useAppSelector(state =>
     selectAssetById(state, foxEthLpAssetId),
   ).precision
+  const [lpTokenPrice, setLpTokenPrice] = useState<string | null>(null)
   const featureFlags = useAppSelector(selectFeatureFlags)
 
+  useEffect(() => {
+    if (
+      // get price if at least one of lp or farming were on
+      (featureFlags.FoxLP || featureFlags.FoxFarming) &&
+      ethAssetPrecision &&
+      ethMarketData.price &&
+      lpAssetPrecision
+    )
+      getLpTokenPrice(ethAssetPrecision, ethMarketData.price, lpAssetPrecision).then(
+        lpTokenPrice => {
+          setLpTokenPrice(lpTokenPrice.toString())
```

Comment:
Can we make `getLpTokenPrice` resolve to a string vs. a `BN` currently? tl;dr serializes better